src/Attachment.tsx

```diff
@@ -26,17 +60,20 @@ const Youtube = (props: {
 }) =>
     <iframe
         type="text/html"
-        src={ new URI()
-            .domain(YOUTUBE_DOMAIN)
-            .subdomain("")
-            .port("")
-            .segment(["embed", props.embedId])
-            .search({
-                "modestbranding": 1,
-                "loop": props.loop ? 1 : 0,
-                "autoplay": props.autoPlay ? 1 : 0
-            })
-            .toString()
+        src={
+            buildUrl(
+                [
+                    "https://",
+                    YOUTUBE_DOMAIN,
+                    "/embed/",
+                     props.embedId
+                ].join(''),
+                {
+                    "modestbranding": 1,
+                    "loop": props.loop ? 1 : 0,
+                    "autoplay": props.autoPlay ? 1 : 0
+                }
```

Comment:
any reason you're quoting these field names here and below? unless necessary I'd prefer to  have them unquoted.